.github/workflows/ci-casper-client-sdk.yml

```diff
@@ -47,52 +47,64 @@ jobs:
         uses: actions/setup-node@5b949b50c3461bbcd5a540b150c368278160234a #v3.4.0
         with:
           node-version: ${{ matrix.node-version }}
-          cache: 'npm'
+          cache: 'yarn'
+
+        # https://github.com/dtinth/setup-github-actions-caching-for-turbo
+      - name: Setup Turbo cache server
+        uses: dtinth/setup-github-actions-caching-for-turbo@e26dc6b0473cc072107f6cb4f6e69d33ac2fc386 # v1.1.0
 
       - name: Install
-        run: npm install
+        run: yarn install
 
       - name: Audits
-        run: npm audit --omit=dev
+        run: yarn npm audit --environment production
 
       - name: Lints
-        run: npm run lint:ci
+        run: yarn lint:ci
 
       - name: Unit Test
-        run: npm run test:unit
+        run: yarn test
+
+      - name: Generate coverage report
+        run: yarn test:coverage
+
+      - name: Upload coverage reports to Codecov
+        uses: codecov/codecov-action@v3
+        env:
+          CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
 
       - name: Test docs build
         run: npm run docs:build
 
       - name: Test build
         run: npm run build
 
-      # test mainnet
-      - name: Get the mainnet version
-        id: get-mainnet-version
-        run: echo "MAINNET_VERSION=$(curl -s -X POST --data-raw '${{ env.VERSION_QUERY }}' -H "Content-Type:application/json" ${{ env.MAINNET_NODE_URL }} | jq -r .result.build_version | cut -d "-" -f 1)" >> "$GITHUB_ENV"
+      # # test mainnet
```

Comment:
<This ius just a nitpick, ignore if you don't agree>
I know that functionally it is the same thing, but could we, instead of commenting these steps out use an if statement that is false?
```
- name: Get the mainnet version
   if ${{ false }}     
```
This should disable execution of the step, but it will still be interpreted so any syntax errors will be caught and we don't have to commit commented code. Also, after the if-statement we could have a comment explaining when this should be re-enabled.

Reply:
I agree with you